### PR TITLE
Use permissionsList KV map in show/edit views

### PR DESCRIPTION
**Pull request by eckdanny-osi**
screen shot 2018-07-31 at 3 14 52 pm
screen shot 2018-07-31 at 3 14 46 pm

### CODE REVIEW

File: app/javascript/selectors/detailSelector.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ export const selectDetailRecords = state => {
 
 export const permissionsList = state => {
   return state.fetchPermissions.permissions
-    ? state.fetchPermissions.permissions.permissions.map(list => ({
-        label: list,
-        value: list,
+    ? state.fetchPermissions.permissions.permissions.map(item => ({
+        label: item.replace(/-rollout$/, ''),
```
Comment (RamuKammagani-osi) [inline app/javascript/selectors/detailSelector.js]: screen shot 2018-07-31 at 3 30 42 pm


@eckdanny-osi  I just cloned your branch and tried to run it. But i see the above error and it breaks the page when i hit add user button on the user list page! Am i missing something??

**Comment by eckdanny-osi** [comment]
This PR should freeze until [COG-327](https://osi-cwds.atlassian.net/browse/COG-327) has shipped new images to docker hub. The java API (upstream dep) will provide `permissionsList: Array<{label: string, value: string}>` when complete. Waiting for those images to become available now

File: app/javascript/selectors/detailSelector.js (hunk 1 above)
Comment (eckdanny-osi) [reply app/javascript/selectors/detailSelector.js]: should be
```diff
- ? state.fetchPermissions.permissions.permissions.map(item => ({
+ ? state.fetchPermissions.permissions.map(item => ({
```
but not addressing until COG-327 puts new images on docker hub

File: app/javascript/views/userDetail/UserDetailShow.jsx [hunk 2]
```diff
@@ -54,7 +67,7 @@ const UserDetailShow = ({ details, onEdit }) => (
             </div>
             <div className="col-md-3">
               <ShowField label="Assigned Permissions">
-                {details.permissions}
+                {renderPermissions(details.permissions, permissionsList)}
```
Comment (Sushmareddy-OSI) [inline app/javascript/views/userDetail/UserDetailShow.jsx]: @eckdanny-osi , I cloned and tested this branch. Noticed that when new user added with permission from {permessionsList} and saved. The assigned permissions are not viewed on show card.

Comment (eckdanny-osi) [reply app/javascript/views/userDetail/UserDetailShow.jsx]: does a subsequent `GET /user/:userId` API request get made when returning to `show`?

Let's avoid managing a cache of mutated resources on the client. If PUT/POSTs don't yield the updated resource in the response body, then the resource should to be fetched with a GET all over again.

**Comment by eckdanny-osi** [comment]
### 👀 Failing Jenkins CI status checks 👀 

If we depend on cognito for authentication, _all_ of our acceptance test will fail. We're in a tough spot rt now because we can't develop locally without a 3rd party network dependency since there is no mock cognito service for docker-compose. Isolation is broken => impacts tests _and_ dev-prod parity 🙁

**Comment by eckdanny-osi** [comment]
Closing in favor of #103

**Pull request closed without merging**